plugins/plugin-svn/che-plugin-svn-ext-shared/src/main/java/org/eclipse/che/plugin/svn/shared/LockRequest.java

```diff
@@ -53,4 +54,22 @@
     LockRequest withForce(boolean force);
 
     void setForce(boolean force);
+
+    /** @return user name for authentication */
+    String getUserName();
+
+    /** Set user name for authentication. */
```

Comment:
I prefer to document only getter methods in DTOs. Because other docs say nothing useful